lib/api/utilities/text_renderer.rb

```diff
@@ -39,7 +39,11 @@ def initialize(text, format: nil, object: nil)
         @text = text
         @format = format
         @object = object
-        @project = object.project if object.respond_to?(:project)
+        if object.respond_to?(:project)
+          @project = object.project
+        else
+          @project = object
```

Comment:
That means you assume that every object that does not have a project **is** a project?

Remember that we are in a dynamic world and code like this will actually work until it breaks ^^

My current favorite option would be to add an `object.is_a?(Project)`, but there might be more ruby-esque solutions...